epub33/core/index.html

```diff
@@ -7672,20 +7672,20 @@ <h5>The <code>par</code> Element</h5>
 										</p>
 									</li>
 								</ul>
-								<p>The <a href="#elemdef-smil-audio"><code>audio</code> element</a> is OPTIONAL only if
-									its sibling <a href="#elemdef-smil-text"><code>text</code> element</a> refers to
-									audio or video media (see <a href="#sec-embedded-media"></a>), or to textual content
-									intended for rendering via <a href="#sec-tts">text-to-speech</a>.</p>
 							</dd>
 						</dl>
 					</section>
 
 					<section id="sec-smil-text-elem">
 						<h5>The <code>text</code> Element</h5>
 
-						<p>The <code>text</code> element references an element in the <a>EPUB Content Document</a>. A
-								<code>text</code> element typically refers to a textual element but can also refer to
-							other EPUB Content Document media elements (see <a href="#sec-embedded-media"></a>).</p>
+						<p>
+							The <code>text</code> element references an element in an <a>EPUB Content Document</a>.
+							A <code>text</code> element typically refers to a textual element but can also refer to
+							other EPUB Content Document media elements (see <a href="#sec-embedded-media"></a>).
+							In the absence of a sibling <code>audio</code> element this element SHOULD refer to a 
```

Comment:
I would remove the last sentence ("In the absence of a sibling...") because it makes it sound like the presence of a sibling `audio` element makes non-TTS-able content accessible, but this is not true. E.g. an image of text alongside an audio rendering of that text still would not allow a user to explore that text with their screenreader, to find out the spelling.

Reply:
I am not sure I agree. That sentence does not say anything about the situation you describe (which is of course perfectly o.k.) and I do not see why it would suggest disallowing it. On the other hand, we have moved the reference to TTS (in case there is no sibling) here from the definition of the `audio` element; if we remove this sentence there is no reference to the usage of TTS any more, which I think is more of a problem.

Reply:
> In the absence of a sibling <code>audio</code> element this element SHOULD refer to a textual content intended for rendering via <a href="#sec-tts">text-to-speech</a>.

Can I read into this that in the presence of a sibling audio element, the text element may refer to non-TTS-able content?

E.g. 

```
<par>
  <text src="page.html#video">
  <audio src="background-music.mp3">
</par>
```

Which is potentially totally inaccessible but still legal (that is ok, this isn't the EPUB a11y spec). 

But as soon as you remove the audio, we are saying with the above sentence that then you need to have
```
<par>
  <text src="page.html#text-not-video">
</par>
```

Because why? Speaking to this specific example, an in-page video in its own `par` could make sense for an author, to initiate playback after some narration. (Even though I would still rather remove the ability to reference embedded media with `text`, that's another discussion).

Reply:
We seem to disagree...

Everything you write above is of course correct, with the exception of:

> But as soon as you remove the audio, we are saying with the above sentence that then you need to have

I read this sentence as if the statement used a _MUST_. But we say _SHOULD_, which essentially say "do this, unless you have a good reason not to".  Your use case of referring to a video element has a use case indeed, which is a good reason to keep it. But I think we can agree that this use case is probably a very minor one compared to the TTS case (I almost said 'in practice', though we do not know how either TTS or embedded videos are implemented 😒). If you really dislike it, we can exchange _SHOULD_ to _MAY_, which weakens the requirement even more.

> (Even though I would still rather remove the ability to reference embedded media with text, that's another discussion)

Yep, I agree. I think the simplification of SMIL into MO went a step too far. I would prefer to re-introduce the SMIL `<ref>` element, and maybe also the `<video>` element. This would make the spec way cleaner and would help with these ambiguities. Do you think it is worth raising this to the WG? Or should we declare this as the boat that has sailed?

Reply:
> Everything you write above is of course correct, with the exception of:
> 
> > But as soon as you remove the audio, we are saying with the above sentence that then you need to have
> 
> I read this sentence as if the statement used a _MUST_. But we say _SHOULD_, which essentially say "do this, unless you have a good reason not to". Your use case of referring to a video element has a use case indeed, which is a good reason to keep it. But I think we can agree that this use case is probably a very minor one compared to the TTS case (I almost said 'in practice', though we do not know how either TTS or embedded videos are implemented 😒). If you really dislike it, we can exchange _SHOULD_ to _MAY_, which weakens the requirement even more.

Well, all of  MO is a _SHOULD_ anyway 🙃

I still don't think that requiring content to be TTS-able only if there is no `audio` makes sense. What problem are we solving?

> 
> Yep, I agree. I think the simplification of SMIL into MO went a step too far. I would prefer to re-introduce the SMIL `<ref>` element, and maybe also the `<video>` element. This would make the spec way cleaner and would help with these ambiguities. Do you think it is worth raising this to the WG? Or should we declare this as the boat that has sailed?

I think that boat is long gone. I would remove the ability to reference to embedded media. Make `text` mean actual text. Maybe add `image` for images? But then you either restrict `src` of image to embedded images or you have to define a viewport for them, which gets complex. 

Introducing `ref` and `video` is tempting but I don't see a practical case for it when it comes to content creation and RS implementation. Not to mention the spec would get considerably more intimidating.

We have a lot of thoughts on this over in [sync-media-pub](https://w3c.github.io/sync-media-pub/).

Reply:
> I would remove the ability to reference to embedded media. Make text mean actual text. Maybe add image for images? But then you either restrict src of image to embedded images or you have to define a viewport for them, which gets complex.

Using `text` for embedded media is wildly unintuitive, to put it mildly, but I'm really worried about the can of worms we'd be opening this late in the revision by changing to much, too.

There's so much about `text` without `audio` for any non-text content that is problematic to implement. We allow images, but we don't define how to determine the text to TTS. Is it only the alt text? Do you account for `aria-describedby`, `aria-label(ledby)`, and `aria-details`, too? We're assuming, I guess, that the reading system will be ARIA-aware and use its rules.

And what is the likelihood we'll find any implementations of audio/video initiation from a `text` element? I suspect that feature will get tagged "under-implemented".

Reply:
> Using text for embedded media is wildly unintuitive, to put it mildly, but I'm really worried about the can of worms we'd be opening this late in the revision by changing too much, too.

Yep. Maybe, in a future version of EPUB, [sync-media-pub](https://w3c.github.io/sync-media-pub/) would take over MO, so touching MO may be unnecessary. Let us be silent on this now.

@mattgarrish could you be the decider on the (mild:-) disagreement @marisademeglio and I have on whether or not to keep the sentence

>  In the absence of a sibling audio element this element SHOULD refer to a textual content intended for rendering via [text-to-speech](https://github.com/w3c/epub-specs/pull/2001#sec-tts).

in the definition of the `<text>` element or not? This is prominently part of the current spec, though in an awkward  manner, and I still believe it is better to keep it as it is in the current PR; @marisademeglio thinks it may be misleading. 

I would like to merge and close this issue...

Reply:
> Well, all of MO is a SHOULD anyway 🙃

Right, exactly... 

> I still don't think that requiring content to be TTS-able only if there is no audio makes sense. What problem are we solving?

That is a valid question but, at this point, my worry is to remove or, more exactly, to strongly downplay a feature that is in the  current spec. We could just make the `<audio>` element REQUIRED, full stop, and remove that feature, but that is not something we should do in this iteration (we are not supposed to make EPUB 3.2 instances invalid 3.3). After all, the only purpose of this PR (and its original issue) is a clean-up.

Reply:
Can we change the last sentence to not contain a normative statement? So like:

"In the absence of a sibling audio element, textual content referred to by this element may be rendered via [text-to-speech](https://pr-preview.s3.amazonaws.com/w3c/epub-specs/pull/2001.html#sec-tts)."

Reply:
That works for me. I have made the change.

Reply:
I mark this issue as resolved; I think the whole discussion around the possible re-introduction of `<ref>` should be put aside for now.